src/extension.ts

```diff
@@ -95,7 +95,6 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 export async function deactivate() {
-	if (telemetry) {
-		await telemetry.dispose();
-	}
+	await telemetry?.dispose();
```

Comment:
I always wonder in such code if first dispose throws then second is never executed.

Reply:
mmm maybe we ensure dispose methods never throws